src/test_sort.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import random
+
+
+EDGE_CASES = [[[100, 100, 100, 100, 100], [100, 100, 100, 100, 100]],
+              [[999999999999999, 0], [0, 999999999999999]],
+              [[1000000000, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 1000000000]],
+              [[], []],
+              [[1, 100, 1, 100, 1, 100, 1, 100, 1, 100],
+               [1, 1, 1, 1, 1, 100, 100, 100, 100, 100]]
+              ]
```

Comment:
Your tests should include some negative numbers just to ensure that it works for more than just 0+